modules/administration/pages/removing-channels.adoc

```diff
@@ -39,103 +33,64 @@ It can be done with {webui} or on the command line.
+mgrctl exec -ti -- spacewalk-remove-channel -l
 ----
 
 
-====  Verify Channel System Subscriptions
+===  Verify Channel System Subscriptions
 
 .Procedure: Verifying Channel System Subscriptions Using {webui}
 
-. In the {productname} {webui}, in the sidebar navigate to menu:Software[Channel List > All].
+. In the {productname} {webui}, navigate to menu:Software[Channel List > All].
 . Locate [literal]``Systems`` column on the right.
 . Check the [literal]``Systems`` column for the channel, or any child channels, that you want to remove.
 
 
-.Procedure: Verify Channel System Subscriptions Using the Command Line
+.Procedure: Verifying Channel System Subscriptions Using the Command Line
 
-. Run the following command:
+. Run the following command on the {productname} container host:
 +
 ----
-spacecmd -- softwarechannel_listsystems <Channel Label>
----- 
+mgrctl exec -ti -- 'spacecmd -- softwarechannel_listsystems <Channel Label>'
+----
 
 
 [[removing-channel]]
-=== Removing Channel
-
-A channel, with its metadata, can be removed in the SUMA command line by using the [literal]``spacewalk-remove-channel`` command. 
-This command does not remove the packages themselves from the filesystem. 
+== Remove Channel
 
-Removing packages from the filesystem is described in <<cleaning-up-channel-packages>>.
+A channel, with its metadata, can be removed with command line tools only.
+The [command]``spacewalk-remove-channel`` command automatically removes the software packages, too.
+It also removes the metadata from the database and the files from the storage medium.
 
 
 
-==== Removing an Individual Channel
+=== Remove a Child Channel
 
-.Procedure: Removing an Individual Channel Using the Command Line
+.Procedure: Removing a Child Channel Using the Command Line
 
-. To remove an individual channel run the following command:
+. To remove an individual channel run the following command on the {productname} container host:
 +
 ----
-spacewalk-remove-channel -c channel-label
+mgrctl exec -ti -- spacewalk-remove-channel -c channel-label
 ----
 +
-. To remove multiple channels at the same time, use [literal]``-c`` flag for each channel, followed by the [literal]``channel-label``. 
+. To remove multiple channels at the same time, use [literal]``-c`` flag for each channel, followed by the [literal]``channel-label``.
   For example:
 +
 ----
-spacewalk-remove-channel -c channel-label1 -c channel-label2
+mgrctl exec -ti -- spacewalk-remove-channel -c channel-label1 -c channel-label2
 ----
 
 
-==== Removing a Parent and All Its Child Channels
+=== Remove a Parent and All Its Child Channels
 
 .Procedure: Removing a Parent and All Its Child Channels
-. It is also possible to remove a parent channel with all of its child channels, like this:
+. To remove a parent channel with all of the child channels run a command such as:
 +
 ----
-spacewalk-remove-channel -a sles12-sp5-pool-x86_64
+mgrctl exec -ti -- spacewalk-remove-channel -a sles12-sp5-pool-x86_64
```

Comment:
You used placeholders like "channel-label" on the command above, but here you use one "real" one.
Is this by a reason?

Reply:
probably previous writer used a rel one because it is a parent channel.  But I changed that one now.